rosidl_generator_cpp/test/test_traits.cpp

```diff
@@ -13,14 +14,230 @@
 // limitations under the License.
 
 #include <gtest/gtest.h>
+// #include "rosidl_generator_cpp/idl/idl_only_types.hpp"
```

Comment:
Because the `test_interface_files` package doesn't export this file atm: https://github.com/ros2/test_interface_files/blob/c4d710e3394ad09a71a257f5490688653fb441b3/CMakeLists.txt#L32

Reply:
It would be nice to have a comment here explaining why.
If not, it's hard to understand the reason.